### PR TITLE
[sync] 2020/11/25

### PR DESCRIPTION
**Pull request by Vadaski**

### CODE REVIEW

File: src/docs/get-started/install/_get-sdk.md [hunk 1]
```diff
@@ -167,6 +167,48 @@ command again to verify that you’ve set everything up correctly.
 当你安装了任一缺失部分的依赖后，
 可以再次运行 `flutter doctor` 命令来确认是否成功安装。
 
+### Downloading straight from GitHub instead of using an archive
+
+### 直接从 Github 上（而不是归档）中下载
```
Comment (AlexV525) [inline src/docs/get-started/install/_get-sdk.md:172]:
```suggestion
### 直接从 Github 上（而不是归档中）下载
```

File: src/docs/get-started/install/_get-sdk.md [hunk 2]
```diff
@@ -167,6 +167,48 @@ command again to verify that you’ve set everything up correctly.
...
+
+_This is only suggested for advanced use cases._
+
+_该建议仅适用于高级用例_
```
Comment (AlexV525) [inline src/docs/get-started/install/_get-sdk.md:176]:
```suggestion
**该建议仅适用于高级用例**
```

File: src/docs/get-started/install/_get-sdk.md [hunk 3]
```diff
@@ -167,6 +167,48 @@ command again to verify that you’ve set everything up correctly.
...
+[更新环境变量][Update your path]，并运行 `flutter doctor`。
+这个命令将会告诉你，是否还缺少运行 Flutter 所需要安装的其他依赖项（例如 Android SDK）。
+
+If you did not use the archive, Flutter will download necessary development binaries as they
+are needed (if you used the archive, they are included in the download). You may wish to
+pre-download these development binaries (for example, you may wish to do this when setting
+up hermetic build environments, or if you only have intermittent network availability). To
+do so, run the following command:
+
+如果你不使用归档，Flutter 将会下载必要的开发二进制文件（如果你使用的归档，那么这些文件已经包含在内了）。你也许会想要提前下载这些开发二进制文件（例如，您可能希望设置系统构建环境，或是您的网络可用性不佳）。
```
Comment (AlexV525) [inline src/docs/get-started/install/_get-sdk.md:200]: 这里换个行~

**Review by AlexV525 — APPROVED** [review]
LGTM modulo comments below.

**Comment by Vadaski** [comment]
done~

**Review by AlexV525 — APPROVED** [review]

**Pull request merged**